src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_format.py

```diff
@@ -65,6 +75,11 @@ def _format_group(item):
     """
     table_info = {_property_map[key]: str(item[key]) for key in item if key in _property_map}
 
+    try:
+        table_info['PLATFORM'] = item['platform']['osType']
+    except(KeyError, TypeError):
+        pass
```

Comment:
NIT: move this to the end, in the same order as in `_order_map`. Also add a space `except (`.

Reply:
Ok